ui/src/elle/interceptor/FetchInterceptor.js

```diff
@@ -0,0 +1,19 @@
+import { errorEmitter, loadingEmitter } from '../../App';
+
+export const fetchInterceptor = async () => {
+
+  const originalFetch = window.fetch;
+
+  window.fetch = async (...args) => {
```

Comment:
window.fetchi üle kirjutada ei ole ilmselt hea idee. Võiks loading emitter aware fetchi siit lihtsalt exportida ja andmeid laadivad kohad sellega asendada.

Reply:
Vaatasin et kõigis esimestes tulemustes fetch interceptori kohta soovitati selliselt läheneda, nt:
https://blog.logrocket.com/intercepting-javascript-fetch-api-requests-responses/
https://dev.to/snigdho611/react-js-interceptors-with-fetch-api-1oei
Aga see mõte tundub ka hea. Proovin, kuidas õnnestub